retropie_packages.sh

```diff
@@ -73,5 +73,5 @@ fi
 printMsgs "console" "${__INFMSGS[@]}"
 
 if [[ -n $__joy2key_pid ]]; then
-    kill -INT $__joy2key_pid 2>/dev/null
+    kill -INT $__joy2key_pid 2>/dev/null || echo "Failed to kill joy2key"
```

Comment:
if this line causes it to stop due to be running with "exit on error" set, there are many other places which will fail - so I think this will need to be disabled via your automation software.